channels/google/email-headers.rst

```diff
@@ -1,6 +1,6 @@
 .. note::
 
-   Email header manipulation in Google channels works in the same way as
+   Header based actions in Google channels works in the same way as
```

Comment:
works -> work

Reply:
Actually, I'm not sure if plural or singular would be correct.

Reply:
Thanks. I think plural would fit better here, I change it.